google-cloud-tools-plugin/src/com/google/gct/login/IntelliJGoogleLoginMessageExtender.java

```diff
@@ -0,0 +1,11 @@
+package com.google.gct.login;
+
+import com.google.gct.idea.util.GctBundle;
+
+public class IntelliJGoogleLoginMessageExtender implements GoogleLoginMessageExtender {
+
+  @Override
+  public String additionalLogoutMessage() {
+    return GctBundle.message("clouddebug.logout.additional.message");
```

Comment:
There's a method to obtain all states listening in the background that is being renamed in an upcoming PR (not yet submitted). I'd rather merge that PR, then extract the method and make it available for this message extender to check if there are any debug sessions running in the background and display the message only if there's any.